src/garage/config.py

```diff
@@ -0,0 +1,6 @@
+"""Config garage project path."""
+import pathlib
+import sys
```

Comment:
Oh -- please don't do this. Appending the PYTHONPATH using sys is brittle and doesn't work in many scenarios. 

If you can guarantee that your script will be called from the root of the repo, the benchmarks package should exist by virtue of being a directory.

Everything which appears in the `garage.` namespace has to be in src/garage, and the fact that it's difficult to import garage packages without installing using some environment manager is actually intentional (it helps us catch bugs which only happen for people who install the package )

How are you planning on running this code? That will inform how to deal with the namespaces.

Reply:
Honestly, I think the more correct way of handling the needs here is to have another `setup.py` in `benchmarks`, that can install a package called `garage_benchmarks` of something. If that were the case, to run the benchmark script you would first do `pip install -e ./benchmarks` from the garage root directory.

It would require moving the files in benchmarks into a subdirectory though, like `benchmarks/garage_benchmarks`.

Reply:
I've implemented what I describe here in #1411.